tracer/test/Datadog.Trace.Tests/Telemetry/config_prefix_block_list.json

```diff
@@ -131,6 +131,8 @@
   "DD_AwsDynamoDb_",
   "DD_TRACE_HardcodedSecret_",
   "DD_HarcodedSecret_",
+  "DD_TRACE_TrustBoundaryViolation_",
+  "DD_TrustBoundaryViolation_",
```

Comment:
You don't need to do this any more, or make a PR to dd-go 🙌 Although... is a test failing without it? 🤔  If so, then leave it in, and I'll fix in a separate PR 🙂 

```suggestion
```

Reply:
I think you replied to the wrong message, but correct it's not necessary any more after this PR  🙂 
- https://github.com/DataDog/dd-trace-dotnet/pull/4872